procedures: starting a new workspace (#2227)
codenvy/che-docs: modules/user-guide/pages/url-parameter-for-the-devfile-file-name.adoc

:_content-type: CONCEPT
:description: URL parameter for the devfile file name
:keywords: devfile, override-devfile-file-name, override-file-name, file-name, devfile-name, different-devfile-name
:navtitle: URL parameter for the devfile file name
// :page-aliases:

[id="url-parameter-for-the-devfile-file-name_{context}"]
= URL parameter for the devfile file name

> When you visit a URL for starting a new workspace, {prod-short} searches the linked Git repository for a devfile with the file name `.devfile.yaml` or `devfile.yaml`. So the devfile in the linked Git repository must follow this convention.

Reviewer: ```suggestion
When you visit a URL for starting a new workspace, {prod-short} searches the linked Git repository for a devfile with the file name `.devfile.yaml` or `devfile.yaml`. Therefore, the devfile in the linked Git repository must follow the convention.
```
